app/Http/Controllers/Backend/User/DashboardController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Http\Controllers\Backend\User;
+
+use App\Enum\StatusVisibility;
+use App\Http\Controllers\Controller;
+use App\Models\Status;
+use App\Models\User;
+use Carbon\Carbon;
+use Illuminate\Contracts\Pagination\Paginator;
+use Illuminate\Database\Eloquent\Builder;
+
+abstract class DashboardController extends Controller
+{
+
+    public static function getPrivateDashboard(User $user): Paginator {
```

Comment:
Do we really need to supply a user here? Isn't a default `auth()->user()` enough?

Reply:
Each user has a different global dashboard based on private profiles and their follows. I think it makes sense to differentiate explicitly.

Reply:
In addition, you can theoretically test better this way.